index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main class="main-wrapper">
+        <div class="first-wrapper">
+                <img src="images/Snapshoot-Portfolio.png" alt="">
+                <div>
+                    <h3>
+                        Tonic
+                    </h3>
+                </div>
+                <div class="list-items">
+                    <h4>CANOPY</h4>
+                    <ul class="ul-list">
+                        <li>Back End Dev</li>
+                    </ul>
+                    <ul class="ul-list">
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="paragraph">
+                    <h4>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h4>
+                </div>
+                <div class="btn">
+                    <button type="button" class="btn-1">html</button>
+                    <button type="button" class="btn-2">css</button>
+                    <button type="button" class="btn-3">javaScript</button>
+                </div>
+                <div class="big-btn">
+                    <button type="button" class="btn-4">See Project</button>
+                </div>
+        </div>
+
+        <!--End of first wrapper ...........................................-->
+
+        <div class="second-wrapper">
+               <img src="images/Snapshoot-Portfolio-2.png" alt="">
+               <div>
+                <h3>
+                    Multi-Post Stories
+                </h3>
+            </div>
+            <div class="list-items">
+                <h4>CANOPY</h4>
+                <ul class="ul-list">
+                    <li>Back End Dev</li>
+                </ul>
+                <ul class="ul-list">
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="paragraph">
+                <h4>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </h4>
+            </div>
+            <div class="btn">
+                <button type="button" class="btn-1">html</button>
+                <button type="button" class="btn-2">css</button>
```

Comment:
- [ ] Kindly use `flexbox` to align content within the project wrapper.

Reply:
@Cathella 

Can you kindly throw more light on this?